nc_test4/Makefile.am

```diff
@@ -41,21 +41,28 @@ if !ISCYGWIN
 NC4_TESTS += tst_h_strbug tst_h_refs
 endif
 
+check_PROGRAMS =
+TESTS =
+
 # Build test programs plus programs used in test scripts.
-check_PROGRAMS = $(NC4_TESTS) renamegroup tst_empty_vlen_unlim
-TESTS = $(NC4_TESTS)
+if USE_NETCDF4
```

Comment:
This if USE_NETCDF4 is not needed. The nc_test4 directory will be skipped if netCDF-4 is not enabled, so there is no need to check USE_NETCDF4, it will always be true.

Reply:
Thanks for pointing out the redundancy; reviewing this now then will get the PR out.

Reply:
Rather than kick off the whole CI chain by removing this redundancy, I'll note it for removal after the merge.